setup.py

```diff
@@ -125,11 +125,18 @@ def get_release_info():
     '{:.5f} = taurus.qt.qtgui.base:floatFormatter',
 ]
 
+config_loaders = [
```

Comment:
For the entry point name, I think it is better to use something that identifies the registrar (e.g., in this case `"taurus.py"`, `"taurus.json"`, `"taurus.xml"`, ...) because it helps for providing a mechanism to select/deselect the plugin (using `taurus.core.util.plugin.selectEntryPoints`).

Reply:
I was not aware of `taurus.core.util.plugin.selectEntryPoint` existence. Will change.